topi/src/topi.cc

```diff
@@ -174,6 +174,31 @@ TVM_REGISTER_GLOBAL("topi.cast")
   *rv = cast(args[0], args[1]);
   });
 
+TVM_REGISTER_GLOBAL("topi.elemwise_sum")
+.set_body([](TVMArgs args, TVMRetValue *rv) {
+  *rv = elemwise_sum(args[0], args[1]);
```

Comment:
Corresponding to above comment, number or args can be obtained inside topi by size().